tests-manual/debugger/setBuffer

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#help 	Set the number of reading to hold in each buffer, passed an integer argument
```

Comment:
Set the number of reading to hold in each buffer, passed an integer argument -> Set the number of readings to hold in each buffer, passing an integer argument